From 5aeaad0f0122ee2ef5207978531c701b31a162cf Mon Sep 17 00:00:00 2001
From: Xu Mao <B34688@freescale.com>
Date: Mon, 30 Dec 2013 17:19:47 +0800
Subject: [PATCH 18/45] ENGR00293494 m2ts format media files can't be
 recognized

add support for displaying m2ts file in Gallery&CactusPlayer

Signed-off-by: Xu Mao <B34688@freescale.com>
---
 media/java/android/media/MediaFile.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/java/android/media/MediaFile.java b/media/java/android/media/MediaFile.java
index bc5f6f0..1bf5524 100644
--- a/media/java/android/media/MediaFile.java
+++ b/media/java/android/media/MediaFile.java
@@ -211,6 +211,7 @@ public class MediaFile {
         addFileType("MKV", FILE_TYPE_MKV, "video/x-matroska");
         addFileType("WEBM", FILE_TYPE_WEBM, "video/webm");
         addFileType("TS", FILE_TYPE_MP2TS, "video/mp2ts");
+        addFileType("M2TS", FILE_TYPE_MP2TS, "video/mp2ts");
         addFileType("AVI", FILE_TYPE_AVI, "video/avi");
 
         if (isWMVEnabled()) {
-- 
1.8.0

